src/java/com/zimbra/graphql/models/inputs/GQLInviteReplyVerbInput.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra GraphQL Extension
+ * Copyright (C) 2018 Synacor, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software Foundation,
+ * version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ * ***** END LICENSE BLOCK *****
+ */
+package com.zimbra.graphql.models.inputs;
+import com.zimbra.common.gql.GqlConstants;
+
+import io.leangen.graphql.annotations.types.GraphQLType;
+
+/**
+ * The GQLInviteReplyVerbInput enum.<br>
+ * Contains verbs for calendar invite reply.
+ * @see com.zimbra.soap.mail.message.SendInviteReplyRequest
+ *
+ * @author Zimbra API Team
+ * @package com.zimbra.graphql.models.inputs
+ * @copyright Copyright © 2018
+ */
+@GraphQLType(name=GqlConstants.CLASS_INVITE_REPLY_VERB_INPUT, description="Verbs for send invite reply.")
+public enum GQLInviteReplyVerbInput {
+    ACCEPT,
```

Comment:
don't we need to annotate values with @GraphQLEnumValue annotation?

Reply:
Done. They're needed in the case of changing the name on the front-end or adding descriptions.